### PR TITLE
Add toggle for camera view in area card

### PR DESCRIPTION
**Pull request by bramkragten**
## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/lovelace/editor/config-elements/hui-area-card-editor.ts [hunk 1]
```diff
@@ -64,6 +71,18 @@ export class HuiAreaCardEditor
           )}
           @value-changed=${this._valueChanged}
         ></ha-area-picker>
+        <ha-formfield
```
Comment (ludeeus) [inline src/panels/lovelace/editor/config-elements/hui-area-card-editor.ts:74]: Should we hide or disable the switch if there are no camera entities in the area?

**Review by zsarnett — DISMISSED** [review]
😄 this is exactly what I have in my local branch but my dev environment blew up. Looks good. I think its probably too expensive to check if entities exist for camera. 

Dont show camera at first if check and camera exist then show camera. we can add helper text if needed

File: src/panels/lovelace/editor/config-elements/hui-area-card-editor.ts [hunk 2]
```diff
@@ -47,6 +50,10 @@ export class HuiAreaCardEditor
     return this._config!.theme || "";
   }
 
+  get _show_camera(): string {
```
Comment (zsarnett) [inline src/panels/lovelace/editor/config-elements/hui-area-card-editor.ts:53]:
```suggestion
  get _show_camera(): boolean {
```

File: src/panels/lovelace/editor/config-elements/hui-area-card-editor.ts (hunk 1 above)
Comment (bramkragten) [reply src/panels/lovelace/editor/config-elements/hui-area-card-editor.ts:74]: Don't think so, if the user later adds a camera to the area it should also work? 🤷‍♂️

**Review by zsarnett — APPROVED** [review]

**Comment by spacegaier** [comment]
@bramkragten  I don't think it is intended to look like this, if the camera is not available?

image

**Pull request merged**